MANIFEST.in

```diff
@@ -1,6 +1,13 @@
 include qiskit/qasm/libs/*.inc
-include qiskit/schemas/*json
+include qiskit/schemas/*.json
 include qiskit/VERSION.txt
-include qiskit/transpiler/passes/routing/cython/stochastic_swap/*pyx
-include qiskit/transpiler/passes/routing/cython/stochastic_swap/*pxd
-recursive-include qiskit/test/mock/backends *json
+include qiskit/transpiler/passes/routing/cython/stochastic_swap/*.pyx
+include qiskit/transpiler/passes/routing/cython/stochastic_swap/*.pxd
+recursive-include qiskit/test/mock/backends *.json
+
+# Include the tests files.
+recursive-include test *.py
```

Comment:
Instead of doing this and L10-L13, you could replace this with just
```
graft test
```
and probably also a exclude to make sure we don't pull in pyc files.

Reply:
I think it is less risky to include rather than exclude - from the description:

> For the non-python files, I went with specifying them in separate entries as the amount of items seems still reasonable, and to minimize the risk of including extraneous files (jupyter notebooks temporary checkpoints come to mind).

We can reevaluate as things evolve.